sklearn/inspection/_permutation_importance.py

```diff
@@ -98,6 +98,32 @@ def permutation_importance(estimator, X, y, scoring=None, n_repeats=5,
     ----------
     .. [BRE] L. Breiman, "Random Forests", Machine Learning, 45(1), 5-32,
              2001. https://doi.org/10.1023/A:1010933404324
+
+    Examples
+    --------
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from sklearn.inspection import permutation_importance
+
+    >>> X = [[1,9,9],[1,9,9],[1,9,9],
+    ...      [0,9,9],[0,9,9],[0,9,9]]
+    >>> y = [1,1,1,0,0,0]
+
+    >>> clf = LogisticRegression().fit(X,y)
+    LogisticRegression(...)
+
+    >>> result = permutation_importance(clf, X, y, n_repeats=10,
+    ... random_state=0)
+    {'importances_mean': array([0.5, 0. , 0. ]),
+    'importances_std': array([0.16666667, 0.        , 0.        ]),
+    'importances': array([[0.33333333, 0.66666667],
+        [0.        , 0.        ],
+        [0.        , 0.        ]])}
```

Comment:
The full dictionary output should not be displayed as the results are stored in the `result` variable. I think showing the values of `result.importances_mean` and `result.importances_std` is enough.